packages/monaco/src/browser/monaco-editor-model.ts

```diff
@@ -279,6 +281,35 @@ export class MonacoEditorModel implements ITextEditorModel, TextEditorDocument {
         return this.model;
     }
 
+    /**
+     * Find all matches in an editor for the given options.
+     * @param options the options for finding matches.
+     *
+     * @returns the list of matches.
+     */
+    findMatches(options: FindMatchesOptions): FindMatch[] {
+        const results: monaco.editor.FindMatch[] = this.model.findMatches(
+            options.searchString,
+            false,
+            options.isRegex,
+            options.matchCase,
+            // eslint-disable-next-line no-null/no-null
+            options.matchWholeWord ? this.editorPreference['editor.wordSeparators'] : null,
```

Comment:
For the fallback value, you'd have to move the `editor.wordSeparators` default out of the schema into an exported const, so that we can import it in this file and use it here.